test/powershell/Language/Scripting/Scripting.Followup.Tests.ps1

```diff
@@ -90,4 +90,43 @@ Describe "Scripting.Followup.Tests" -Tags "CI" {
 
         TestFunc2 | Should -Be @("1", "2")
     }
+
+    It "'[NullString]::Value' should be treated as string type when resolving .NET method" {
+
+        $testType = 'NullStringTest' -as [type]
+        if (-not $testType) {
```

Comment:
What is the scenario when this branch is not taken? Leftovers from another test? How can we make sure that another test defines `NullStringTest` exactly how we want? Removing the type in `AfterEach` is more reasonable to me.

Reply:
When running the test multiple times within the same PowerShell session, the type would have already been declared, and calling `Add-Type` again will result in an error.
The type is made available by `Add-Type`, and thus cannot be removed until the current PS process exits.